nacc_utils/nacc_automation/sel.py

```diff
@@ -89,7 +89,7 @@ def main(argv):
         str = "https://"+username+":"+password+"@www.alz.washington.edu/MEMBER/sitesub.htm"
         driver.get(str)
         # Get the 1Florida ADRC project url
-        nacc_project_elem = driver.find_element_by_xpath("/html/body/table[2]/tbody/tr[1]/td[2]/font/ul[1]/li[5]/a")
+        nacc_project_elem = driver.find_element_by_xpath("/html/body/table[2]/tbody/tr[1]/td[2]/font/ul[1]/li[6]/a")
```

Comment:
Is there a way to make this more robust? Specifically, can we select the `a` element that contains "1Florida ADRC" instead of selecting the 6th one?

Reply:
I'll have to do some selenium research but I think there should be a way to check this. I'll look into it.

Reply:
4a471fe now looks for 1Florida ADRC by name.

Reply:
Talked offline; @naomidb tested the changes by doing a full run and confirmed correctness.